boards/arm/nrf5340dk_nrf5340/Kconfig

```diff
@@ -56,6 +56,10 @@ config BOARD_ENABLE_DCDC_HV
 	select SOC_DCDC_NRF53X_HV
 	default y
 
+config BOARD_ENABLE_128MHZ_APP
+	bool "Enable 128MHz speed in application core"
+	select SOC_CLOCK_128MHZ_NRF53X_APP
```

Comment:
I think this option should be removed (and such prompt should be instead used in the `SOC_CLOCK_128MHZ_NRF53X_APP` option). Unlike the DC/DC converters, 128 MHz clock can be enabled without a need of any additional components present on a board, so the option should be always available for the application core in nRF5340, regardless of the board in which the SoC is used.